### PR TITLE
Translation chapter #1 and #2

### PR DESCRIPTION
**Pull request by josepaulolima**

### CODE REVIEW

**Comment by rafaelbcerri** [comment]
@JulianaAmoasei can you review it?

File: 04-social-network/00-rwd/00-opening/README.md [hunk 1]
```diff
@@ -1,48 +1,36 @@
-# Opening: La web desde un móvil
+# Abertura: A Web em um Dispositivo Móvel
 
-- Tipo: `lectura`
-- Formato: `self-paced`
-- Duración: `15min`
+- Tipo: `leitura`
+- Formato: `ritmo próprio`
```
Comment (rafaelbcerri) [inline 04-social-network/00-rwd/00-opening/README.md]: @josepaulolima `self-paced` foi traduzido como `individual` lá no [course-parser](https://github.com/Laboratoria/course-parser/blob/master/lib/part.js#L49), se você puder alterar aqui e nos outros arquivos também =)

**Comment by josepaulolima** [comment]
I made a new commit changing "self-paced" to "individual". 
Finish module 2.

**Comment by ivandevp** [comment]
@rafaelbcerri and @JulianaAmoasei your review is pending in order to accept this PR :smiley:

**Comment by rafaelbcerri** [comment]
Everything seems fine for me =)
@ivandevp can you deploy it only in staging, we have to validate something first?

**Comment by ivandevp** [comment]
Cool! I'm going to accept the PR and deploy it on staging.

**Pull request merged**